scripts/joinmarket-qt.py

```diff
@@ -103,6 +104,22 @@ def update_config_for_gui():
             jm_single().config.set("GUI", gcn, gcv)
 
 
+def switchToBIP78Payjoin(parent, endpoint_url):
```

Comment:
Are these two functions not more naturally methods of the Spend tab?

Reply:
Yes, together with `checkAddress()`.

Reply:
And 'checkAmount()` too.